packages/table/src/regions.ts

```diff
@@ -89,12 +89,21 @@ export type ICellInterval = [number, number];
 export type ICellCoordinate = [number, number];
 
 /**
- * A ZERO-indexed region of cells.
- *
  * @see `Regions.getRegionCardinality` for more about the format of this object.
  */
 export interface IRegion {
+    /**
+     * The first and last row indices in the region, inclusive and zero-indexed.
+     * If `rows` is `null`, then all rows are understood to be included in the
+     * region.
+     */
     rows?: ICellInterval;
+
+    /**
+     * The first and last column indices in the region, inclusive and
+     * zero-indexed. If `cols` is `null`, then all columns are understood to be
+     * included in the region.
+     */
     cols?: ICellInterval;
```

Comment:
perhaps add `| null` to these types to match docs? or does it support `undefined` (cuz of optional prop)?

I know we don't use `sNC` in blueprint but these annotations are useful in the outside world.

Reply:
maybe the type should be `cols: ICellInterval | null`? not optional, just nullable?

Reply:
Totally agree, but I'm hesitant to make that change in an otherwise docs-only PR. Might break people who are manually constructing their regions if they happened to pass `undefined`.

Reply:
Added `| null` but left the `?` for now to keep allowing `undefined`.